plugins/action/action.h

```diff
@@ -119,6 +119,13 @@ class Action
     Result return_to_launch() const;
 
     /**
+     * @brief Send command to *change flight mode* (FW).
```

Comment:
Please describe what this does for an outsider that doesn't know the concept of flight modes and neither the acronym FW.

Reply:
And also comment that this is only applicable for hybrid/VTOL vehicles.